### DISCUSSION TITLE
how can I haw multi auth bearer in different documents?

### DISCUSSION THREAD

**Discussion by sohrab-jan** [DarkaOnLine/L5-Swagger#517] [Q&A]
i have generated authentication with jwt but when i logout from one document , the other  documents logout two.

* @OA\SecurityScheme(
     *     securityScheme="bearerSellerAuth",
     *     in="header",
     *     name="bearerAuth",
     *     type="http",
     *     scheme="bearer",
     *     bearerFormat="JWT",
     *     )

     this is my code for auth

**Comment by DarkaOnLine** [comment]
Have you tried `persist_authorization` option: https://github.com/DarkaOnLine/L5-Swagger/blob/master/config/l5-swagger.php#L277

**Comment by sohrab-jan** [comment]
thanks for your useful package
i have several rule in my project as module
and i want to each module have different document

i done it with change in config file

do you have some tutorial for the package?
